views/partials/profile/bids.handlebars

```diff
@@ -4,7 +4,7 @@
     <div class="card horizontal hoverable">
 
       <div class="card-image">
-        <img src="/css/Images/home.jpeg">
+        <img src="{{this.dataValues.Home.imgUrl}}">
       </div>
 
```

Comment:
this looks find for now with imgUrl. I am trying to move that to a another table with an association with Home